src/Commands/Site/InfoCommand.php

```diff
@@ -46,4 +47,38 @@ public function info($site)
     {
         return $this->getPropertyList($this->sites->get($site));
     }
+
+    /**
+     * @param TerminusModel $model A model with data to extract
+     * @return PropertyList A PropertyList-type object with applied filters
+     */
+    public function getPropertyList(TerminusModel $model)
+    {
+        $data = $model->serialize();
+        if ($this->input()->getOption('format') === 'table') {
+            // Manipulate framework to make it more user-friendly.
+            if (isset($data['framework'])) {
+                $data['framework'] = $this->getFrameworkFriendlyName($data['framework']);
```

Comment:
Don't change the `framework` field here; if you do this, then json, yaml & similar formats will also be changed. We want these "raw" formats to continue to return `drupal`, `drupal8`, etc.

To change the value only for table format, see "addDatetimeRenderer", and make a new cell renderer for the framework field.

Reply:
Thanks! Updated.